apps/client/src/pages/login/page/login.tsx

```diff
@@ -0,0 +1,102 @@
+import { Header } from '@confeti/design-system';
+import {
+  BtnDeleteBlack20,
+  IcKakao,
+  IcApple,
+  ImgTypelogoBig,
+  Confeti3DLogo21,
+} from '@confeti/design-system/icons';
+import { Button } from '@confeti/design-system';
+import * as styles from './login.css';
+import { routePath } from '@shared/constants/path';
+
+const DESCRIPTION_TEXT =
+  '가입 시, confeti의\n[이용약관] 및 [개인정보처리방침]에 동의하게 돼요.';
+const LINK_MAP: LinkMap = {
+  이용약관: routePath.PRIVACY_CONFETI,
+  개인정보처리방침: routePath.PRIVACY_PERSONAL,
+};
+
+interface LinkMap {
+  [key: string]: string;
+}
+
+const parseLinkContent = (
+  part: string,
+  lineIndex: number,
+  index: number,
+): JSX.Element | string => {
+  const bracketMatch = part.match(/\[(.*?)\]/);
+  if (bracketMatch) {
+    const content = bracketMatch[1];
+    const link = LINK_MAP[content];
+
+    if (link) {
+      return (
+        <a
+          key={`${lineIndex}-${index}-${content}`}
+          href={link}
+          className={styles.atagText}
+          target="_blank"
+          rel="noreferrer"
+        >
+          {content}
+        </a>
+      );
+    }
+  }
+  return part;
+};
+
+const processLine = (line: string, lineIndex: number): JSX.Element => {
+  const parts = line.split(/(\[.*?\])/);
+  const processedParts = parts.map((part, index) =>
+    parseLinkContent(part, lineIndex, index),
+  );
+
+  return <p key={`line-${lineIndex}`}>{processedParts}</p>;
+};
+
+const parseDescription = (text: string): (JSX.Element | string)[] | null => {
+  if (!text) return null;
+
+  const lines = text.split('\n');
+  return lines.map((line, lineIndex) => processLine(line, lineIndex));
```

Comment:
```suggestion
  return text.split('\n').map(processLine);
```
`text.split('\n')`을 `lines` 변수에 할당하지 않고 바로 `map()`을 호출하면 불필요한 변수 선언이 줄어들어 코드가 간결해질 거 같은데 혹시 의도가 있어서 변수로 할당한 걸까요?

Reply:
혹시 텍스트의 \n을 html에도 줄바꿈으로 인식시키고 싶으시면 `white-space: pre-line`라는 css 속성 이용하시면 됩니다!